ui/src/app/applications/components/pod-logs-viewer/pod-logs-viewer.scss

```diff
@@ -6,18 +6,12 @@
     font-size: 14px;
     font-family: monospace;
     background-color: white;
-    @include themify($themes){
-        background-color: themed('background-2');
-    }
     padding: 0;
     color: black;
-    @include themify($themes){
-        color: themed('text-2');
-    }
 
     &--inverted {
-        background-color: black;
-        color: white;
+        background-color: #303237;
+        color: #fff;
```

Comment:
Hi @surajyadav1108 , just one nit, since fff is white, could we leave this as `white` to be consistent with line 8 above, and what we had before?

It's very subtle to me, but what's the reason for using 303237 instead of black?  Do you know if this is what the yaml editor uses?

I'll merge your PR after your response.

Reply:
I see it - it's dark-theme-background-2

Reply:
Yes, I just wanted to keep it as dark-theme-background-2 because the tint black felt a little too contrasting. but, I have changed it back to how it was, with a black background and white text. Now you can merge it.

Reply:
and then #18470